app/controllers/amf_controller.rb

```diff
@@ -883,12 +883,10 @@ def tags_ok(tags)
   # in the +tags+ hash.
   def strip_non_xml_chars(tags)
     new_tags = {}
-    unless tags.nil?
-      tags.each do |k, v|
-        new_k = k.delete "\000-\037\ufffe\uffff", "^\011\012\015"
-        new_v = v.delete "\000-\037\ufffe\uffff", "^\011\012\015"
-        new_tags[new_k] = new_v
-      end
+    tags&.each do |k, v|
```

Comment:
This is the only one of these that I feel itchy scratchy about... Do we know what this code will do if `tags` is nil? Perhaps this one is worth leaving alone.

Reply:
It behaves the same:

```
irb(main):002:0> nil&.each
=> nil
```

Reply:
I agree it's odd looking.  We can use a guard clause instead.

Reply:
Hmm, lets leave it as is for now and see how this ends up looking when it makes its way upstream. Thanks for checking!